.licenseignore

```diff
@@ -5,6 +5,7 @@
 .swift-format
 .github/*
 *.md
+**/*.config
```

Comment:
This is to fix the license checking CI job, since it checked and failed on the new file not having a license header

Reply:
Ah, thank you!